### ISSUE TITLE
[Quest] Becoming a spellfire Tailor

### ISSUE THREAD

**Issue by Kaporalhart** [dalaranwow/dalaran-wow#3426]
**Description**: Currently on my level 80 mage, i can't complete the quest becaause the item doesn't spawn the mob you're supposed to kill. I tried running around in all the area (this is supposed to work anywhere within netherstorm) but nope.

**Current behaviour**: At Netherstorm, upon using the "nether-wraith beacon" after the 1.5 seconds casting and the arcane explosion-like animation, nothing happens.

**Expected behaviour**: At Netherstorm, upon using the "nether-wraith beacon", 1 to 2 nether wraiths should spawn (according to wowhead).

**Steps to reproduce the problem**:

1. Go to netherstorm with the quest
2. Use the item anywhere
3. Observe that nothing happens.

http://www.wowhead.com/quest=10832/becoming-a-spellfire-tailor

**Comment by ImprezaWoW** [comment]
It was disabled intentionally due to some abusings, needs fix so you couldn't spam-spawn the mob for XP.

**Comment by Rushor** [comment]
i don't have any problems to summon them http://i.imgur.com/oBnwYba.jpg


maybe the creature fall under the ground, make shure to use it on a save position

but these is an exploit, you can summon as much creatures as possible

6b1d09b78ae3379b6a65a2d2b9d72faca91e7166

i added a check, that there will be summoned no creatures, if there is already a creature with entry 22408 within 30 yards to the caster

**Comment by ImprezaWoW** [comment]
Until it's fixed, GMs complete the quest in-game, open a ticket.

**Comment by Golganeth** [comment]
Needs polishments. Two Furious Nether-Wraith spawn instantly when you use the quest item in Area-52. They spawn instantly and not after the cast of the item is done. According to sources it should be just one NPC and after the cast is over.

> Minor issue, no longer an exploit

**Comment by t9xnos** [comment]
Am having the same issue , what to do ? :( Ive tried every spor peole said , nothing .

**Comment by AgeAgainst** [comment]
3 Feb, 2017 4:51am server time

I have tried this on two different characters on two different accounts.  Multiple places in Netherstorm including inside of Area 52.  I also spent a good minute flying around, landing every five seconds so I was further than the 30 yards from the check and did it again.

The quest is not working anymore.  The mob that is supposed to spawn simply no longer spawns anymore.
